guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -1,10 +1,10 @@
 [id="Adding_a_Baseboard_Management_Controller_Interface_{context}"]
 = Adding a baseboard management controller (BMC) interface
 
+{Project} can control the power status of bare metal hosts.
 Use this procedure to configure a baseboard management controller (BMC) interface for a host that supports this feature.
 
 .Prerequisites
-* The `ipmitool` package is installed.
 * You know the MAC address, IP address, and other details of the BMC interface on the host, and the appropriate credentials for that interface.
```

Comment:
```suggestion
* You know the MAC address, IP address, and other details of the BMC interface on the host, and authentication credentials for that interface.
```
Would this make it clearer what the appropriate credentials are? (Non blocker)

Reply:
Yes, applied.